tests/src/lv_test_conf.h

```diff
@@ -104,10 +104,6 @@ typedef void * lv_user_data_t;
 #define  LV_DPI_DEF         130
 #endif
 
-#if defined(LVGL_CI_USING_SYS_HEAP)
-#undef LV_USE_FLOAT
```

Comment:
@kisvegabor 
Why does this [commit ](https://github.com/lvgl/lvgl/commit/84c8cf8573e41cd969375cf8937e4399ee85b846) require `#undef LV_USE_FLOAT`?

Reply:
Just run a config without floats too.

Reply:
I added a mandatory dependency of `LV_USE_FLOAT` to `LV_USE_MATRIX` (see [here](https://github.com/lvgl/lvgl/pull/4883#issuecomment-2184946203)), and I found that only by removing this `#undef LV_USE_FLOAT` compilation test can it pass properly. Can I delete this `#undef LV_USE_FLOAT`?

Reply:
Okay, let's delete it for now. We need to restructure tests later anyway.